src/py42/clients/securitydata.py

```diff
@@ -33,145 +28,10 @@ def savedsearches(self):
         """A collection of methods related to retrieving forensic search data.
 
         Returns:
-            :class: `py42._internal.services.securitydata.SavedSearchService`
+            :class: `py42._internal.services.SavedSearchService`
```

Comment:
I believe this should be `py42._internal.services.savedsearch.SavedSearchService`